src/test/resources/features/ifs/SupressionByPostCode.feature

```diff
@@ -82,13 +82,13 @@ Feature: Suppression by Postcode
       | 35                   | 5520                 |
     And the 1st debt summary will contain
       | numberChargeableDays | interestDueDailyAccrual | totalAmountIntDuty |
-      | 155                   | 35                      | 502314             |
+      | 155                  | 35                      | 505520             |
```

Comment:
I don't _think_ these failed on the pipeline, but they did fail locally. Please run them yourself to check. Also not sure which IFS rules caused the numbers to change.